bcda/cclf/cclf.go

```diff
@@ -152,26 +149,55 @@ func importCCLF0(ctx context.Context, fileMetadata *cclfFileMetadata) (map[strin
 }
 
 func importCCLF8(ctx context.Context, fileMetadata *cclfFileMetadata) (err error) {
-	db := database.Connection
-
-	repository := postgres.NewRepository(db)
-
 	if fileMetadata == nil {
```

Comment:
Can we remove this logic here? I think we can guarantee that fileMetadata is not nil at this point.

Reply:
https://github.com/CMSgov/bcda-app/blob/1dee98bd451adfeb0287f16b8ef454a7e0039e9d/bcda/cclf/cclf.go#L335